content/millennium/r4/foundation/other/health-cards.md

```diff
@@ -70,11 +70,11 @@ Represents the patient and the clinical data related to a laboratory test result
 
 Contains the following resources:
 
-* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-dm.html#tab-ms) profile with the following fields, if valued:
+* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://hl7.org/fhir/uv/shc-vaccination/2021Sep/) profile with the following fields, if valued:
   * [Patient official name](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.name){:target="_blank"}
   * [Date of Birth](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.birthDate){:target="_blank"}
 
-* One or more Observation resources that follow the [InfectiousDiseaseLaboratoryResultObservationDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-infectious-disease-laboratory-result-observation-dm.html) profile with the following fields, if valued:
+* One or more Observation resources that follow the [InfectiousDiseaseLaboratoryResultObservationDM](http://hl7.org/fhir/uv/shc-vaccination/2021Sep/) profile with the following fields, if valued:
```

Comment:
should be to the specific lab results data minimization profile http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-infectious-disease-laboratory-result-observation-dm.html

Reply:
> should be to the specific lab results data minimization profile http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-infectious-disease-laboratory-result-observation-dm.html

Fixed in commit https://github.com/cerner/fhir.cerner.com/pull/874/commits/6891a9e92a9faf1db82a825354e912180f12c6af